`make manpages` is broken after vendor update

```
$ make manpages
mkdir -p dist/manpages/
ls rkt/*.go | grep -vE '_test.go|main.go|_gen.go' | GO15VENDOREXPERIMENT=1 GOARCH="amd64" CGO_ENABLED=1 CC="gcc" CXX="g++" GOPATH="/home/ubuntu/rkt/src/github.com/coreos/rkt/build-rkt-1.9.1+git/gopath" GOROOT="/usr/local/go" GOARCH=amd64 CC= xargs "go" run rkt/manpages_gen.go
rkt/image_cmd_common.go:24:2: cannot find package "github.com/appc/spec/discovery" in any of:
	/usr/local/go/src/github.com/appc/spec/discovery (from $GOROOT)
	/home/ubuntu/rkt/src/github.com/coreos/rkt/build-rkt-1.9.1+git/gopath/src/github.com/appc/spec/discovery (from $GOPATH)
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -33,7 +33,7 @@ script:
  -      if [ "${TARGET}" == "amd64" ]; then
                 ./autogen.sh;
                 ./configure --with-stage1-flavors=host,fly --enable-tpm=auto;
-                make unit-check;
+                make unit-check manpages bash-completion;
         elif [ "${TARGET}" == "arm64" ]; then
                 eval "$(GIMME_ARCH=${TARGET} GIMME_CGO_ENABLED=1 ./gimme.local ${TRAVIS_GO_VERSION})";
                 ./autogen.sh;

--- a/rkt/rkt.mk
+++ b/rkt/rkt.mk
@@ -32,12 +32,16 @@ include makelib/build_go_bin.mk
 manpages: GO_ENV += GOARCH=$(GOARCH_FOR_BUILD) CC=$(CC_FOR_BUILD)
 manpages: | $(GOPATH_TO_CREATE)/src/$(REPO_PATH)
 	mkdir -p dist/manpages/
-	ls rkt/*.go | grep -vE '_test.go|main.go|_gen.go' | $(GO_ENV) xargs "$(GO)" run rkt/manpages_gen.go
+	ls $(GOPATH_TO_CREATE)/src/$(REPO_PATH)/rkt/*.go | \
+		grep -vE '_test.go|main.go|_gen.go' | \
+		$(GO_ENV) xargs "$(GO)" run $(GOPATH_TO_CREATE)/src/$(REPO_PATH)/rkt/manpages_gen.go
 
 bash-completion: GO_ENV += GOARCH=$(GOARCH_FOR_BUILD) CC=$(CC_FOR_BUILD)
 bash-completion: | $(GOPATH_TO_CREATE)/src/$(REPO_PATH)
 	mkdir -p dist/bash_completion/
-	ls rkt/*.go | grep -vE '_test.go|main.go|_gen.go' | $(GO_ENV) xargs "$(GO)" run rkt/bash_completion_gen.go
+	ls $(GOPATH_TO_CREATE)/src/$(REPO_PATH)/rkt/*.go | \
+		grep -vE '_test.go|main.go|_gen.go' | \
+		$(GO_ENV) xargs "$(GO)" run $(GOPATH_TO_CREATE)/src/$(REPO_PATH)/rkt/bash_completion_gen.go
 
 protobuf:
 	scripts/genproto.sh

--- a/tests/build-and-run-tests.sh
+++ b/tests/build-and-run-tests.sh
@@ -160,6 +160,7 @@ function build {
     CORES=$(grep -c ^processor /proc/cpuinfo)
     echo "Running make with ${CORES} threads"
     make "-j${CORES}"
+    make manpages bash-completion
 
     if [[ ${PRECLEANUP} == true ]]; then
         rm -rf "${BUILD_DIR}/tmp/usr_from_${RKT_STAGE1_USR_FROM}"

